packages/block-library/src/post-author/block.json

```diff
@@ -66,5 +66,6 @@
 			"clientNavigation": true
 		}
 	},
+	"editorStyle": "wp-block-post-author-editor",
```

Comment:
Why is this change needed?

Reply:
It's necessary because we're readding the editor style file and we want it to be taken into consideration when previewing the blocks in the editor.